.github/workflows/scheduled-releases.yml

```diff
@@ -0,0 +1,51 @@
+name: scheduled-releases
+
+on:
+  schedule:
+    # Runs every Wednesday at 08:00 UTC (midnight PST / 1:00 AM PDT)
+    - cron: '0 8 * * 3'
+
+  # Allows manual triggering for convenience
+  workflow_dispatch:
+
+jobs:
+  weekly-release:
+    runs-on: ubuntu-latest
+
+    env:
+      # Declare to satisfy linter, will be set dynamically during workflow execution
+      PR_NUMBER: ""
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Configure Git
+        run: |
+          git config user.name 'github-actions[bot]'
+          git config user.email 'github-actions[bot]@users.noreply.github.com'
+
+      - name: Create weekly release PR
+        shell: pwsh
+        run: |
+          # Create PR from main to rel/weekly, capture created URL
+          $PR_URL = $(gh pr create `
+            --base rel/weekly `
+            --head main `
+            --title "Scheduled weekly release $(Get-Date -Format 'yyyy-MM-dd')" `
+            --body "Automated weekly release PR created by scheduled workflow." `
+            --fill-verbose `
+            --reviewer michael-hawker)
```

Comment:
Can we use an existing team here like `ToolkitTeam`? Or leave it blank?

I guess that's the catch, is we need someone to approve this PR, eh?

Reply:
That's correct. I coded it to request you since you normally need to hit the nuget gate approval anyway, but anyone with write perms can approve as-is and we could tweak branch protection to accommodate this any time.